syncwatcher.go

```diff
@@ -459,6 +450,23 @@ func relativePath(path string, folderPath string) string {
 	return path
 }
 
+// Returns a function to test whether a file should be ignored
+// Ignorefile must contain the absolute path to ".stignore". The returned
+// function takes the path of the file to be tested relative to its folders root
+func createIgnoreTest(folderPath string) func(relPath string) bool {
+	ignores := ignore.New(false)
+	ignores.Load(filepath.Join(folderPath, ".stignore"))
+	return func(relPath string) bool {
+		if strings.HasPrefix(relPath, versionsFolder) { return true }
```

Comment:
Ok, if I would know either Go or the code that would really healp: With `os.PathSeparator` respectively the already defined variable `pathSeparator` mentioned by you it is now either
`strings.HasPrefix(relPath, versionsFolder + pathSeparator)`
or with your method
`strings.SplitN(relPath, pathSeparator, 2)[0] == versionFolder`.
I would actually prefer the first method, it seems easier to understand to me.